wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+[Extended Chat Console wikilink]: /wiki/Chat_Console/ "more info can be found on Chat Console under Extended Chat Console"
+
+<!-- image -->
+[Replay Auto]: ./Replay_Auto.jpg "Replay with Auto, offline and online account play respectively"
+[Replay discussion]: ./Replay_discussion.jpg "Discussion button options during a replay"
+[Replay discussion comment]: ./Replay_discussion_comment.jpg "Making a comment"
+[Replay discussion example]: ./Replay_discussion_example.jpg "An example of comment flow"
+[Replay Solo]: ./Replay_Solo.jpg "A local replay"
+[Replay failed]: ./Replay_failed.jpg "Left is a regular game over screen and right is a failed replay game over screen"
+[Replay Multi]: ./Replay_Multi.jpg "Left is the original Multi play and right is an exported Multi replay"
+[Replay Multi fail]: ./Replay_Multi_fail.jpg "A failed Multi replay will still be played."
+[Replay Auto on]: ./Replay_Auto_on.jpg "Replay with Auto when connected to Bancho"
+[Replay server]: ./Replay_server.jpg "Left is downloading the server replay and right is the replay in question."
+[Replay preview result]: ./Replay_preview_result.jpg "An example of preview gameplay's result screen"
+
+# Replay
+
+## Introduction
+
+Replay is a direct emulation of a play, either by a player or a perfect play like [Auto][Auto wikilink] mod.
+As long as the local replay emulation file with a file extension, `.osr`, was available with the difficulty/beatmap it reference or a top _Global leaderboard_ play was downloaded, a complete replay can be constructed by osu! to be watched.
+
+## Replay options
+
+![Replay with Auto, offline and online account play respectively][Replay Auto]
+
+_Left is offline replay; Right is online replay_
+
+To rewatch a replay, hold the custom default grave/tilde (`` `/~ ``) key (can be changed in `Options` sidebar; inside _Change keyboard bindings_ as **Quick Retry (hold briefly)**) or `Ctrl`+`R` for a while.
+The screen should darken and a sound will be played when the quick restart was successful.
+Releasing the key early will fail the restart.
+
+Access to [Chat Console][Chat Console wikilink](`F8`)/[Extended Chat Console][Extended Chat Console wikilink](`F9`) will still be available.
+Press the `Tab` key to hide the currently-selected leaderboard.
+Press the `H` key to hide all elements of replay except the game mods used and **UNRANKED** notice if played by [Auto][Auto wikilink].
+
+[Cinema][Cinema wikilink] replay will hide all the replay options, gameplay access, and all the skin elements of the game mode.
+Also, the mod will enable the background image/video, the pass storyboard, 0% background dim, and hit sounds.
+To skip the starting/ending point of the play, press the `Spacebar` key (the long horizontal bar in the keyboard).
+
+### Discussion
+
+This button **only appears when logged-in with an osu! account connected to an online Bancho server and the difficulty/beatmap has been uploaded to the osu! server**.
```

Comment:
Attempt:-

```Markdown
The button **only appears when an ingame osu! account was connected to online Bancho server and the difficulty/beatmap can be found in the osu! beatmaplist (ranked or otherwise)**
```

Both conditions must be satisfied.